packages/elements/src/datetime-picker/index.ts

```diff
@@ -314,42 +332,12 @@ export class DatetimePicker extends ControlElement implements MultiValue {
   @property({ type: Boolean, attribute: 'show-seconds', reflect: true })
   public showSeconds = false;
 
-  private _placeholder = '';
-  /**
-   * Placeholder to display when no value is set
-   * @param placeholder Placeholder
-   * @default -
-   */
-  @property({ type: String })
-  public set placeholder(placeholder: string) {
-    const oldPlaceholder = this._placeholder;
-    if (oldPlaceholder !== placeholder) {
-      this._placeholder = placeholder;
-      this.requestUpdate('placeholder', oldPlaceholder);
-    }
-  }
-  public get placeholder(): string {
-    return this._placeholder || this.format;
-  }
-
   /**
    * Toggles the opened state of the list
    */
   @property({ type: Boolean, reflect: true })
   public opened = false;
 
-  /**
-   * Set error state
-   */
-  @property({ type: Boolean, reflect: true })
```

Comment:
Missing jsdoc for `error` & `warning` prop.

Reply:
Fixed